spec/controllers/tech_blog_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+describe TechBlogController, type: :request do
```

Comment:
Given that this is a request spec, perhaps we should change the name of the spec file, and this `describe` as well